README.md

```diff
@@ -1,16 +1,14 @@
-![Build docker base image (dev)](https://github.com/outflanknl/RedELK/workflows/Build%20docker%20base%20image%20(dev)/badge.svg?branch=maindev)
-![Build docker elasticsearch image (dev)](https://github.com/outflanknl/RedELK/workflows/Build%20docker%20elasticsearch%20image%20(dev)/badge.svg?branch=maindev)
-![Build docker jupyter image (dev)](https://github.com/outflanknl/RedELK/workflows/Build%20docker%20jupyter%20image%20(dev)/badge.svg?branch=maindev)
-![Build docker kibana image (dev)](https://github.com/outflanknl/RedELK/workflows/Build%20docker%20kibana%20image%20(dev)/badge.svg?branch=maindev)
-![Build docker logstash image (dev)](https://github.com/outflanknl/RedELK/workflows/Build%20docker%20logstash%20image%20(dev)/badge.svg?branch=maindev)
+[![Build and push containers](https://github.com/fastlorenzo/RedELK/actions/workflows/containers-build-push.yml/badge.svg)](https://github.com/fastlorenzo/RedELK/actions/workflows/containers-build-push.yml)
```

Comment:
This should be `outflanknl/RedELK`

Reply:
@fastlorenzo thoughts?

Reply:
yes, makes sense, bad copy paste when testing, good catch!